### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by colourmeamused**

### CODE REVIEW

**Comment by colourmeamused** [comment]
@batmat Will leave this with you to make a call on what you want to do, you can revert to the asciidoc version if you want.
- I did follow the pull request model and assigned you for the code changes, and released a beta first. Merged and released 0.5 after no feedback for over two weeks since the plugin was in a broken state.
- README and Changelog updates were an afterthought, both were out of date and neglected. Since it was a documentation change only, I just made the change rather than create yet another pull request and wait for two weeks.
- Apologies for breaking the wiki link, I have updated it. Your call on how to take things forward from here.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,14 @@
-= Changes for the _Jenkins Parameterized Scheduler Plugin_
+## Changes for the _Parameterized Scheduler Plugin_
 
-* 0.1 : First release to the world
-* 0.2 : Not user facing change
-* 0.4-beta-1 Pipeline support Beta [JENKINS-37139](https://issues.jenkins-ci.org/browse/JENKINS-37139)
-* 0.4 Pipeline support
-* 0.5 Added [Declarative pipeline syntax support](https://github.com/jenkinsci/parameterized-scheduler-plugin#declarative-pipeline-configuration-example) and [JENKINS-44031](https://issues.jenkins-ci.org/browse/JENKINS-44031) resolved.
+0.1. First release to the world
```
Comment (batmat) [inline CHANGELOG.md]: Not sure what you are trying to do here? Those should be bullet points, why do you want to transform these into paragraphs?

Comment (batmat) [reply CHANGELOG.md]: Or make them `###` possibly, one entry per version?

Comment (batmat) [reply CHANGELOG.md]: @colourmeamused friendly ping ^

**Comment by res0nance** [comment]
Closing this as the changelog is being moved to github releases. There is a couple years worth of undocumented changes at this stage. The move to github releases does alleviate the issue of changelogs being lost.

**Pull request closed without merging**